### PR TITLE
[Payments] Analytics env var review

### PR DESCRIPTION
**Pull request by peschina**
### Ticket:

- [23328](https://dev.azure.com/OGCIO-Digital-Services/Digital%20Services%20Programme/_workitems/edit/23328)

### Description

Review analytics related environment variables

## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [ ] **New feature**
- [x] **Dev change**
- [ ] **Additional tests**
- [ ] **Documentation**
- [ ] **Other**

### Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] This change might impact the developer experience of others and should be communicated

### Screenshots:

N/A

### CODE REVIEW

File: apps/payments-api/.env.sample [hunk 1]
```diff
@@ -24,18 +24,12 @@ JOURNEY_BACKEND_URL=http://localhost:8009
...
-AUTH_APP_SECRET=auth_app_secret
+LOGTO_M2M_ANALYTICS_ORGANIZATION_ID=ogcio
+LOGTO_M2M_ANALYTICS_SCOPES==
+LOGTO_M2M_ANALYTICS_APP_ID=f5a25fc246b1dbf4a903x
+LOGTO_M2M_ANALYTICS_APP_SECRET=m2m_analytics_local_secret
 
 # Analytics
-ANALYTICS_WEBSITE_ID=website_id
-ANALYTICS_URL=http://localhost:8057
+ANALYTICS_WEBSITE_ID=2c144c54-e581-2591-c5fc-2d9db0b2bbed
```
Comment (alfonsograziano) [inline apps/payments-api/.env.sample:33]: Where is this website id coming? Shall we add a comment just for reference? Is it a valid website id used on dev?

File: apps/payments/.env.sample [hunk 2]
```diff
@@ -30,12 +30,12 @@ NEXT_PUBLIC_MATOMO_PROTOCOL=https
 
 LOGTO_M2M_ANALYTICS_ORGANIZATION_ID=ogcio
 LOGTO_M2M_ANALYTICS_APP_ID=f5a25fc246b1dbf4a903x
-LOGTO_M2M_ANALYTICS_APP_SECRET=m2m_intg_analytics_local_secret
+LOGTO_M2M_ANALYTICS_APP_SECRET=m2m_analytics_local_secret
 
 ANALYTICS_URL=http://localhost:8075
-ANALYTICS_WEBSITE_ID=website_id
+ANALYTICS_WEBSITE_ID=2c144c54-e581-2591-c5fc-2d9db0b2bbed
```
Comment (alfonsograziano) [inline apps/payments/.env.sample:36]: Same question here :)

**Review by alfonsograziano — DISMISSED** [review]
LGTM - just left a question around the value of the website id

File: apps/payments-api/.env.sample (hunk 1 above)
Comment (nnorbert) [reply apps/payments-api/.env.sample:33]: This ID is provided by @andreacappadona17 Your website has to be registered into Matomo and this is the ID Matomo assigned to your website. Based on this ID Matomo will identify our service.

Comment (alfonsograziano) [reply apps/payments-api/.env.sample:33]: Got it, thanks :D

File: apps/payments-api/.env.sample [hunk 3]
```diff
@@ -24,18 +24,12 @@ JOURNEY_BACKEND_URL=http://localhost:8009
...
 STRIPE_WEBHOOK_SECRET=stripe_webhook_secret_key
 
-# Auth envs
-AUTH_OIDC_ENDPOINT=http://localhost:3301/oidc
-AUTH_ORGANIZATION_ID=ogcio
-AUTH_SCOPES=
-AUTH_APP_ID=auth_app_id
-AUTH_APP_SECRET=auth_app_secret
+LOGTO_M2M_ANALYTICS_ORGANIZATION_ID=ogcio
+LOGTO_M2M_ANALYTICS_SCOPES==
```
Comment (andreacappadona17) [inline apps/payments-api/.env.sample:28]:
```suggestion
LOGTO_M2M_ANALYTICS_SCOPES=analytics:website:tracking
```

**Review by andreacappadona17 — APPROVED** [review]

**Pull request merged**